src/test/java/org/folio/rest/impl/RtacResourceImplTest.java

```diff
@@ -350,6 +374,53 @@ public final void testGetRtacByIdNoDueDate() {
     logger.info("Test done");
   }
 
+  @Test
+  public final void testGetRtacByIdVolumeFormatting() {
+    logger.info("Testing for proper volume stirng formatting");
```

Comment:
typo in "stirng"

Reply:
I think "stirng" is an alternative spelling for "string"... 😉

Fixed!